R/use-bbi.R

```diff
@@ -18,40 +22,60 @@ use_bbi <- function(.dir = "/data/apps"){
 
   this_os <- os[sapply(os,grepl,x=R.version$os)]
 
+  if(.version == "latest") {
+    .bbi_url <- current_release(owner = 'metrumresearchgroup', repo = 'babylon', os = "linux")
+  } else {
+    if (this_os == "linux") {
+      .bbi_url <- as.character(glue("https://github.com/metrumresearchgroup/babylon/releases/download/{.version}/bbi_linux_amd64.tar.gz"))
+    } else {
+      warning(glue("User passed `.version = {.version}` and but this argument is only valid on Linux and current OS is {this_os}. Latest version of bbi will be installed."))
```

Comment:
because we don't use the url for the other OS's, we use homebrew for Mac and nothing for Windows. That is something we should also change on the refactor branch ideally, though I'm not sure it's worth implementing Windows right now.

Reply:
Agreed, we can handle in a broader refactor.